src/components/RoadmapList/RoadmapList.module.css

```diff
@@ -141,8 +109,13 @@
   .listWrapper {
     margin: 80px 0 40px;
     gap: 24px;
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(2, 1fr);
+  }
+
+  .listItem {
+    padding: 12px 20px;
+    border: 1px solid var(--primary);
+    border-radius: 8px;
+    flex-basis: calc((100% - 48px) / 3);
   }
```

Comment:
You write listItem class twice? I think you need to change some lines instead of writing class from the beginning

Reply:
fix